pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1123,3 +1156,51 @@ func AllTargets(cd jobspb.ChangefeedDetails) (targets []jobspb.ChangefeedTargetS
 	}
 	return
 }
+
+func logChangefeedCreateTelemetry(ctx context.Context, opts map[string]string, jr *jobs.Record) {
+	jobEventDetails := eventpb.CommonJobEventDetails{
+		JobID:       int64(jr.JobID),
+		JobType:     jobspb.TypeChangefeed.String(),
+		Description: jr.Description,
+		Status:      string(jobs.StatusPending),
+	}
+
+	changefeedDetails := jr.Details.(jobspb.ChangefeedDetails)
+	createChangefeedEvent := &eventpb.CreateChangefeedQuery{
+		CommonJobEventDetails: jobEventDetails,
+		NumTables:             int32(len(AllTargets(changefeedDetails))),
+		SinkType:              "core",
```

Comment:
Perhaps empty string for core? I'm okay either way... it's just we're using "" elsewhere.